### PR TITLE
Added location capabilities (supported editions) for Instance Pools 

### PR DESCRIPTION
**Pull request by rohands**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by rohands** [comment]
@jamestao @johnpaulkee
Would be of great help if you could take a look at this PR!

**Comment by johnpaulkee** [comment]
As discussed offline,
Update Readme.md to reference 2018-06-01-preview capabilities file you added.
Use common Sku object, similar to 2017-10-01-preview

File: specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/capabilities.json [hunk 1]
```diff
@@ -0,0 +1,1285 @@
...
+          },
+          "readOnly": true
+        },
+        "performanceLevel": {
+          "$ref": "#/definitions/PerformanceLevelCapability",
+          "description": "The performance level.",
+          "readOnly": true
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
```
Comment (johnpaulkee) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/capabilities.json]: use common referenced Sku

File: specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/capabilities.json [hunk 2]
```diff
@@ -0,0 +1,1285 @@
...
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "PerformanceLevelUnit",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Sku": {
```
Comment (johnpaulkee) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/capabilities.json]: You can delete this

**Review by johnpaulkee — APPROVED** [review]
I've given my approval with some suggestions

File: specification/sql/resource-manager/readme.md [hunk 3]
```diff
@@ -108,7 +110,6 @@ input-file:
 - Microsoft.Sql/preview/2017-03-01-preview/managedInstanceAdministrators.json
 - Microsoft.Sql/preview/2017-10-01-preview/cancelOperations.json
 - Microsoft.Sql/preview/2017-10-01-preview/cancelPoolOperations.json
-- Microsoft.Sql/preview/2017-10-01-preview/capabilities.json
```
Comment (anthony-c-martin) [inline specification/sql/resource-manager/readme.md]: What's the reason for removing this?

Comment (rohands) [reply specification/sql/resource-manager/readme.md]: So that location capabilities are now referenced to the updated swagger json in 2018-06-01-preview

**Review by anthony-c-martin — APPROVED** [review]

**Comment by johnpaulkee** [comment]
@xccc-msft now that we have ARM sign off, we'll await your approval and submission for merge!
Thanks :)

**Review by xseeseesee — APPROVED** [review]

**Pull request merged**